scipy/sparse/csgraph/tests/test_graph_laplacian.py

```diff
@@ -9,6 +10,16 @@
 from scipy.sparse import csgraph
 
 
+def test_laplacian_value_error():
```

Comment:
As you did bellow, this could also benefit from some parametrize.

Reply:
I do not actually understand what this does, so have not changed it, just moved for better code readability. The goal of this PR is only to add `inplace` and some tests for this, not to beautify/re-write all unit tests.

Reply:
Ok you can leave this out. I am not a fan of this test as well because I don't know what `ValueError` this is referring to (does not have a match statement).

Reply:
Maybe it checks for lacking the required input.